joladnijo/settings.py

```diff
@@ -76,9 +77,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.mysql',
-        'NAME': 'joladnijo',
-        'USER': 'joladnijo',
+        'ENGINE': os.environ.get('DBENGINE', ''),
+        'NAME': os.environ.get('DBNAME', ''),
+        'USER': os.environ.get('DBUSER', ''),
+        'PASSWORD': os.environ.get('DBPASSWORD', ''),
+        'HOST': os.environ.get('DBHOST', ''),
+        'PORT': os.environ.get('DBPORT', ''),
```

Comment:
Itt is a default 3306 megadhato.
```suggestion
        'PORT': os.environ.get('DBPORT', '3306'),
```